server/routes/creatingLicences/crdLicenceInHardStop.test.ts

```diff
@@ -316,6 +316,7 @@ describe('createLicenceRoutes', () => {
         } as AdditionalConditionAp)
         return request(app)
           .post('/licence/create/id/1/check-your-answers')
+          .send({ code: 'condition1' })
```

Comment:
mmh.. this shouldn't be needed but this should still validate.. 

When you submit to check-your-answers it should prevent submitting if there's some additional condition that hasn't been completed.. can you check this still works with your change?